DESCRIPTION

```diff
@@ -26,14 +26,23 @@ URL: https://github.com/sckott/cowsay
 BugReports: https://github.com/sckott/cowsay/issues
 LazyData: yes
 VignetteBuilder: knitr
-Imports:
+Imports: 
+    crayon,
+    curl,
```

Comment:
Curious why all these new dependencies were added?

most seem like the ones you have in `multicolor`. perhaps you thought they needed to be here too?

Reply:
i think we just need: 

```
Imports: 
    crayon,
    fortunes,
    rmsfact
Suggests:
    jsonlite,
    knitr,
    multicolor,
    rmarkdown,
    roxygen2 (>= 6.0.1),
    testthat
```

Reply:
I was getting a travis error in the vignettes when curl wasn't listed as an import. I'll try removing it and see if the error still happens

Reply:
Oh yeah and we definitely don't need all the tidyverse stuff, that should have been taken out

Reply:
Here's the curl error https://travis-ci.org/aedobbyn/cowsay/jobs/399838048#L1263 😕